FirebaseSwiftUI/FirebaseAuthSwiftUI/Sources/Services/AuthService.swift

```diff
@@ -167,21 +173,37 @@ public final class AuthService {
     }
   }
 
+  public func handleAutoUpgradeAnonymousUser(credentials credentials: AuthCredential) async throws {
```

Comment:
No need to duplicate the argument label if you don't want to use a different name inside the function. This can just be

`public func handleAutoUpgradeAnonymousUser(credentials: AuthCredential) async throws {`